packages/framework/package.json

```diff
@@ -101,6 +101,7 @@
     "typescript": "^5.3.3",
     "vitest": "^1.2.1",
     "zod": "^3.23.0",
+    "json-schema-to-ts": "^3.0.0",
```

Comment:
Is there any reason to keep it in the `dependencies` section?

Reply:
We need to include it in `dependencies` as JSON-schema is the "default", barebones validation tool we're using in the framework. This ensures that Dev's can use the lib without installing anything else.